docs/swaggerBase.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 x-api-id: hid-api
 info:
-  version: 3.0.5
+  version: <<<VERSION>>>
```

Comment:
this got way out of date in the last couple weeks. Since we were already generating the docs at deploy-time, I figured why not keep the version in sync automatically too?